qpulseaudioengine.cpp

```diff
@@ -312,29 +335,43 @@ void QPulseAudioEngineWorker::sinkInfoCallback(const pa_sink_info *info)
         return;
 
     /* Now to decide which output to be used, depending on the active mode */
+    qDebug("Deciding output...");
     if (m_audiomode & AudioModeEarpiece) {
         preferred = earpiece;
         audiomodetoset = AudioModeEarpiece;
+        qDebug("Prefer AudioModeEarpiece");
     }
     if (m_audiomode & AudioModeSpeaker) {
         preferred = speaker;
         audiomodetoset = AudioModeSpeaker;
+        qDebug("Prefer AudioModeSpeaker");
     }
     if ((m_audiomode & AudioModeWiredHeadset) && (modes.contains(AudioModeWiredHeadset))) {
         preferred = wired_headset ? wired_headset : wired_headphone;
         audiomodetoset = AudioModeWiredHeadset;
+        qDebug("Prefer AudioModeWiredHeadset");
     }
     if (m_callstatus == CallRinging && speaker_and_wired_headphone) {
         preferred = speaker_and_wired_headphone;
+        audiomodetoset = AudioModeWiredOrSpeaker;
+        qDebug("Prefer AudioModeWiredOrSpeaker");
     }
     if ((m_audiomode & AudioModeBluetooth) && (modes.contains(AudioModeBluetooth))) {
         preferred = bluetooth_sco;
         audiomodetoset = AudioModeBluetooth;
+        qDebug("Prefer AudioModeBluetooth");
```

Comment:
Will the logging be supressed in production builds? Otherwise I suggest removing it.

Reply:
I really need this info for when I'm helping porters debug their stuff and they might not know how to build a new package with debug-level logging enabled.